.vib/jenkins/goss/goss.yaml

```diff
@@ -0,0 +1,32 @@
+command:
+  jenkins-cli:
+    exec: java -jar /bitnami/jenkins/home/war/WEB-INF/lib/cli-${APP_VERSION}.jar -s http://localhost:8080 -auth ${JENKINS_USERNAME}:${JENKINS_PASSWORD} help
+    exit-status: 0
+    stdout: []
+    stderr: []
+    timeout: 20000
+  jenkins-cli-user:
```

Comment:
Although I like this test, we have 2 user related checks and not a lot of Jenkins related tests. Do you think there is something else valuable to test in the Jenkins pod?

Reply:
It is true that we have two user-related tests, but their scope is completely different. One checks the user running the Jenkins process inside the container and the other the jenkins user used to log-in in to the CLI.

I personally would prefer to remove the `jenkins-cli` check instead of the `jenkins-cli-user` one.

------

> Do you think there is something else valuable to test in the Jenkins pod?

Yes, the CLI offers a lot more commands (e.g. `list-jobs`, `create-job`, `create-node`, ...), but they are difficult to test. For instance, `list-jobs` would return an empty list; `create-job` and `create-node` both need to provide an XML definition of the resource. Do you think it is worth it?

Reply:
I don't think it's worth it. I'm not sure about this but I also think that Jenkins CLI needs to be installed in a separate step since it doesn't come by default.
I'd say to leave this test as it is.

Reply:
The Jenkins CLI jar comes by default in the container. It is located at `/bitnami/jenkins/home/war/WEB-INF/lib/cli-${APP_VERSION}.jar`.

> I'd say to leave this test as it is.
So do I. The rest commands do not seem to be easy to test/maintain.